docs/source/10-minutes-to-dask.rst

```diff
@@ -391,6 +392,7 @@ triggering computation, we can inspect the task graph to figure out what's going
          >>> b.visualize()
 
       .. image:: images/10_minutes_array_graph.png
+         :alt: Two neighboring blocks with thin strips near their shared border representing the shared data.
```

Comment:
Is this alt text referring to a different image? It doesn't seem like it goes with this one.

I was expecting something like "Dask task graph of ... blah blah blah" for alt text belonging to `images/10_minutes_array_graph.png`

Reply:
Yes, this very definitely does not go with this image. I've added an alt-text suggestion for the actual image here below.